subsys/llext/Kconfig

```diff
@@ -62,6 +62,7 @@ config LLEXT_SHELL_MAX_SIZE
 
 config LLEXT_STORAGE_WRITABLE
 	bool "llext storage is writable"
+	default y if XTENSA
```

Comment:
This doesn't look quite right, xtensa could easily be loading from non-writable storage as well. The architecture has nothing to do with the Kconfig imo.